.browserslistrc

```diff
@@ -0,0 +1,15 @@
+# the minSdk specified in the config.xml is 22, which maps to android 5.0
```

Comment:
Start comments with capital letter, and end with a period. That's usually what the Google style says.